src/QsCompiler/QirGeneration/DebugInfoManager.cs

```diff
@@ -52,16 +68,24 @@ private static string GetQirProducerIdent()
             return compilationInfo.Name + " with " + qirGenerationInfo.Name + " V " + qirGenerationInfo.Version;
         }
 
+// SECTION: Exposed member variables
```

Comment:
In general the code in this repo does not explicitly marks sections. When this is done, it is usually in large files and using `#region` and `#endregion`. Since this is still a small file, I would recommend not explicitly marking sections but if you do, I would recommend using `#region` and `#endregion`.

Reply:
The sections are mostly just for me while I'm developing in this file. I added a TODO note to remove the sections before merging to main. Is that ok to do?

Reply:
Yes, that's ok for this PR.